### PR TITLE
use price notifier

### PR DESCRIPTION
**Pull request by sstanculeanu**

### CODE REVIEW

File: cmd/priceFeeder/config/config.toml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+[Eth]
```
Comment (iulianpascalau) [inline cmd/priceFeeder/config/config.toml:1]: can we simplify this file to use only the relevant config values?
we should have a dedicated config struct

File: cmd/priceFeeder/main.go [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/blockchain"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/builders"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/core/polling"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/data"
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/interactors"
+)
+
+const (
+	configPath                = "config/config.toml"
+	base                      = "ETH"
```
Comment (iulianpascalau) [inline cmd/priceFeeder/main.go:27]: we should get these const values from the configs:
we should have a "general" config part for the poll interval, auto send interval, contract address, network path and such and a price fetcher part that will be an array of structs holding the L27-L31 values

File: go.sum [hunk 3]
```diff
@@ -70,6 +70,12 @@ github.com/ElrondNetwork/elrond-go-logger v1.0.6 h1:FGbO2C/s9cfNHag9jaKSB+sjuWUo
 github.com/ElrondNetwork/elrond-go-logger v1.0.6/go.mod h1:cBfgx0ST/CJx8jrxJSC5aiSrvkGzcnF7sK06RD8mFxQ=
 github.com/ElrondNetwork/elrond-sdk-erdgo v1.0.13-0.20211228154855-89564efcb4ad h1:nvqhPJ4taGcYZNX7pGCPrxVXllxsu+A3O6VAgmzKT8U=
 github.com/ElrondNetwork/elrond-sdk-erdgo v1.0.13-0.20211228154855-89564efcb4ad/go.mod h1:JrMdurPigkj7yZtbnLsgAxxPesjFhYtQki2TuofUkr4=
+github.com/ElrondNetwork/elrond-sdk-erdgo v1.0.13-0.20220104122958-3fd284139ec5 h1:6HNkZaxIqDKEDiyQmGwEnJlSsKutl9wdq2NASJvj/VI=
```
Comment (iulianpascalau) [inline go.sum:73]: please run a `go mod tidy` command to remove old go.sum values

File: cmd/priceFeeder/keys/oracle.pem [hunk 4]
```diff
@@ -0,0 +1,5 @@
+-----BEGIN PRIVATE KEY for erd198a4ret7j8sskfkpt6ynjcj7xynlhr2473q5ychjqptp5sye7h9syk7mp5-----
+NmFhY2I5ZGNlYTBhN2NkNmU2MTNkMjMxOTZiNTQ3YTNhODMyZGQzYTc5YmZkN2I5
+ZDIzY2FlNDBhZTYzYTM0ZjI5ZmI1MWU1N2U5MWUxMGIyNmMxNWU4OTM5NjI1ZTMx
+MjdmYjhkNTVmNDQxNDI2MmYyMDA1NjFhNDA5OWY1Y2I=
+-----END PRIVATE KEY for erd198a4ret7j8sskfkpt6ynjcj7xynlhr2473q5ychjqptp5sye7h9syk7mp5-----
```
Comment (iulianpascalau) [inline cmd/priceFeeder/keys/oracle.pem:5]: Maybe a new line character?

**Review by iulianpascalau — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by dragos-rebegea — APPROVED** [review]

**Pull request merged**